sonic-utilities-tests/sku_create_test.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import os
+import pytest
+import swsssdk
+import subprocess
+import filecmp
+import re
+import time
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "scripts")
+input_path = os.path.join(modules_path, "sonic-utilities-tests/sku_create_input")
+sku_def_file = os.path.join(input_path, "Mellanox-SN2700-D48C8.xml")
+sku_create_script = os.path.join(scripts_path, "sonic_sku_create.py")
+output_file_path = os.path.join(modules_path, "sonic-utilities-tests/sku_create_input/Mellanox-SN2700-D48C8_NEW/port_config.ini")
+model_file_path = os.path.join(modules_path, "sonic-utilities-tests/sku_create_input/Mellanox-SN2700-D48C8/port_config.ini")
+
+sys.path.insert(0, test_path)
+sys.path.insert(0, modules_path)
+
+class TestSkuCreate(object):
+    @classmethod
+    def setup_class(cls):
```

Comment:
@madhanmellanox do we need teardown section?

Reply:
Addressed in next commit.